### PR TITLE
fixed permission issue with AWS-EFS backend [BT-445]

### PR DESCRIPTION
**Pull request by nvanaja**

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]

**Comment by nvanaja** [comment]
@kshakir  @aednichols  Could  you review/approve this PR? It affects only AWS EFS backend. We are blocked by this.

**Review by kshakir — COMMENTED** [review]
This change looks safe to me but before merging:
- As the owners of this code has someone from the BT team(?) cloned this and run it through whatever test(s) are in Travis and/or Jenkins?
- Mainly out of curiosity: any idea if the whole AWS backend stopped working where/when/what broke? For example: did the recent dependency upgrades in 68 break something the existing test(s) didn't catch? There wasn't much background in the ticket as to why this fix was suddenly needed, so again just curious.

**Comment by nvanaja** [comment]
> This change looks safe to me but before merging:
> 
> * As the owners of this code has someone from the BT team(?) cloned this and run it through whatever test(s) are in Travis and/or Jenkins?
Are there some special tests needs to be run for this? If so, no. The tests that were run with the build(travis) passed.
> * Mainly out of curiosity: any idea if the whole AWS backend stopped working where/when/what broke? For example: did the recent dependency upgrades in 68 break something the existing test(s) didn't catch? There wasn't much background in the ticket as to why this fix was suddenly needed, so again just curious.
  On EFS backend, the script for each cromwell task  gave permission denied error before this fix. It's nothing to do with 68 dependency upgrade. This is  caused by changes made for CROM-6682. It affects only the AWS-EFS backend.

**Review by jgainerdewar — APPROVED** [review]

**Pull request merged**
